### PR TITLE
Router and VUEx changes for Offline mode - Tasks/9870

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#9870](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9870)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Updated routes and vuex store

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [ ] Functional Tests Updated
* [x] Not Required

### CODE REVIEW

File: Apps/WebClient/src/appsettings.Development.json [hunk 1]
```diff
@@ -35,6 +35,12 @@
             "Comment": true,
             "CovidLabResults": true,
             "Dependent": true
```
Comment (sslaws) [inline Apps/WebClient/src/appsettings.Development.json:37]: Nice catch

File: Apps/WebClient/src/Server/Models/OfflineModeConfiguration.cs [hunk 2]
```diff
@@ -44,6 +44,6 @@ public class OfflineModeConfiguration
         /// <summary>
         /// Gets or sets the list of IPs that can use the webclient during an outage.
         /// </summary>
-        public IEnumerable<string> Whitelist { get; set; } = Array.Empty<string>();
+        public IList<string> Whitelist { get; set; } = new List<string>();
```
Comment (sslaws) [inline Apps/WebClient/src/Server/Models/OfflineModeConfiguration.cs:47]: Why did you change this?  We don’t really need anything more than an enumerator snd the initialization is cheaper the old way.

**Review by sslaws — CHANGES_REQUESTED** [review]

File: Apps/WebClient/src/Server/Models/OfflineModeConfiguration.cs (hunk 2 above)
Comment (FuriousLlama) [reply Apps/WebClient/src/Server/Models/OfflineModeConfiguration.cs:47]: The reason I changed it is that it wasn't serializing the configuration correctly. Making it a less abstract collection worked better

**Review by sslaws — APPROVED** [review]

**Review by leobcgov — APPROVED** [review]
